pkg/minikube/machine/start.go

```diff
@@ -279,6 +281,16 @@ func acquireMachinesLock(name string) (mutex.Releaser, error) {
 	return r, err
 }
 
+func releaseMachineLock(releaser mutex.Releaser, name string, start time.Time) {
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
Please do not call recover() outside of the `main` package. I don't think calling recover() here is the behavior we are looking for. If so, I would rather see the system redesigned.